src/grisp_gpio.erl

```diff
@@ -273,6 +273,18 @@ pin(grisp2, gpio1_7)        -> #{path => <<"/pmod-gpio\0">>,             propert
 pin(grisp2, gpio1_8)        -> #{path => <<"/pmod-gpio\0">>,             property => <<"grisp,gpios\0">>, index => 5};
 pin(grisp2, gpio1_9)        -> #{path => <<"/pmod-gpio\0">>,             property => <<"grisp,gpios\0">>, index => 6};
 pin(grisp2, gpio1_10)       -> #{path => <<"/pmod-gpio\0">>,             property => <<"grisp,gpios\0">>, index => 7};
+%
+% enabled uart pins sendod row (lower)
```

Comment:
These comments are a bit redundant since the atoms already contain `uart` and `i2c` in their names.